var/spack/repos/builtin/packages/py-atlalign/package.py

```diff
@@ -12,6 +12,8 @@ class PyAtlalign(PythonPackage):
     homepage = "https://pypi.org/project/atlalign/"
     url = "https://files.pythonhosted.org/packages/74/b8/4382a758e4ccf5de26e35fb2b33863aea27b99e31e1ce9b4bc1e6bb593ef/atlalign-0.6.0.tar.gz"
 
+    # The first version below just serves to trigger a rebuild!
+    version('0.6.0.20210902', sha256='ecd74c89ecdca0a115252fa662e41b272dff2cdc3ea0d459daa5b7a672070b7c')
```

Comment:
With this version `0.6.0.20210902`, I think that spack fails to fetch the package because it would try to download https://files.pythonhosted.org/packages/74/b8/4382a758e4ccf5de26e35fb2b33863aea27b99e31e1ce9b4bc1e6bb593ef/atlalign-0.6.0.20210902.tar.gz
If it's temporary, you can try with
```suggestion
    version(
        '0.6.0.20210902',
        sha256='ecd74c89ecdca0a115252fa662e41b272dff2cdc3ea0d459daa5b7a672070b7c',
        url='https://files.pythonhosted.org/packages/74/b8/4382a758e4ccf5de26e35fb2b33863aea27b99e31e1ce9b4bc1e6bb593ef/atlalign-0.6.0.tar.gz'
        )
```

Reply:
@GianlucaFicarelli Many thanks for stepping in. You made me realize that, indeed, I messed up with urls. For the re-build hack to work as in `py-tensorflow-addons/package.py`, it suffices to add `url=url,  expand=False` to the parameters of `version`. I completely forgot that. What you suggest is, of course, also valid, but more verbose.

Many thanks again for  your hint. I have made changes in the next commit accordingly.